packages/main/src/themes/sap_horizon/CheckBox-parameters.css

```diff
@@ -1,13 +1,35 @@
 @import "../base/CheckBox-parameters.css";
 
 :root {
-	--_ui5_checkbox_wrapper_padding: .6875rem;
+	--_ui5_checkbox_wrapper_padding: .5rem;
 	--_ui5_checkbox_width_height: 2.75rem;
-	--_ui5_checkbox_inner_border: .0625rem solid var(--sapField_BorderColor);
+	--_ui5_checkbox_inner_border: 0.188rem solid var(--sapNeutralColor);
```

Comment:
0.1875 will result in 3px;
if you are using 0.188 it will result in 3 and some numbers after the pointing which could leads to wrong rounding